manifests/v/VPNetwork/TorGuard/4.8.13/VPNetwork.TorGuard.installer.yaml

```
@@ -13,16 +13,14 @@ InstallModes:
 - silent
 UpgradeBehavior: install
 AppsAndFeaturesEntries:
-- Publisher: '"VPNetwork LLC"'
-  DisplayName: VPNetwork LLC - TorGuard - Online Privacy Protection Services
-  DisplayVersion: '"4.8.13-build.198.2+g9824883"'
-  InstallerType: nullsoft
+- DisplayName: VPNetwork LLC - TorGuard - Online Privacy Protection Services
+  DisplayVersion: "4.8.13-build.198.2+g9824883"
+  Publisher: "VPNetwork LLC"
```

```suggestion
  Publisher: '"VPNetwork LLC"'
```
in YAML `"VPNetwork LLC"` is parsed as `VPNetwork LLC`

It should be parsed as `"VPNetwork LLC"` to match Control Panel so this needs to have single quotes outside of the double quotes.